src/libraries/Microsoft.Extensions.Configuration.Binder/gen/ConfigurationBindingGenerator.Parser.cs

```diff
@@ -506,6 +506,42 @@ private TypeSpec CreateEnumerableSpec(TypeParseInfo typeParseInfo)
                 };
             }
 
+            private bool ConvertTo(ITypeSymbol source, ITypeSymbol dest)
+            {
+                var conversion = _typeSymbols.Compilation.ClassifyConversion(source, dest);
```

Comment:
Nit: type name?

Reply:
Can you clarify more? are you asking to rename the arguments?

Reply:
Replacing `var` with the explicit type name.

Reply:
ah, got it.

Reply:
done.